sdk/core/azure_core/src/http/pipeline.rs

```diff
@@ -176,6 +211,33 @@ impl Pipeline {
             Ok(result)
         }
     }
+
+    /// Sends a [`Request`](http::Request) through each configured [`Policy`] to get a [`BufResponse`](http::BufResponse) that is processed by each policy in reverse.
+    ///
+    /// # Arguments
+    /// * `ctx` - The context for the `Request`.
+    /// * `request` - The `Request` to send.
+    /// * `options` - Options for sending the `Request`, including check success options. If none, [`check_success`] will not be called.
+    ///
+    /// # Returns
+    ///
+    /// A [`http::RawResponse`] if the request was successful, or an `Error` if it failed.
+    /// If the response status code indicates an HTTP error, the function will attempt to parse the error response
+    /// body into an `ErrorResponse` and include it in the `Error`.
+    pub async fn stream(
+        &self,
+        ctx: &http::Context<'_>,
+        request: &mut http::Request,
+        options: Option<PipelineStreamOptions>,
+    ) -> crate::Result<http::BufResponse> {
+        let (core_stream_options, send_options) = options.unwrap_or_default().deconstruct();
```

Comment:
I believe that the 2nd tuple returned is a `stream_options`, not a `send_options`